libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -875,11 +877,14 @@ def add_version_header(tc):
             "name": "__cpp_lib_print",
             "values": {"c++23": 202207},
             "headers": ["ostream", "print"],
+            # TODO: Are we missing something here?
```

Comment:
@mordante

Reply:
This matches https://isocpp.org/std/standing-documents/sd-6-sg10-feature-test-recommendations#__cpp_lib_print 
The other paper affecting print https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p2539r4.html does not change a FTM. What do you think we might miss?

Reply:
I was wondering whether we were missing e.g. `_LIBCPP_HAS_NO_LOCALIZATION` or something like that.